### PR TITLE
Update CampaignTest.java

### PR DESCRIPTION
**Pull request by RAGHURD1**
ADDED COMMENT

### CODE REVIEW

File: src/test/java/com/crm/Campaign/CampaignTest.java [hunk 1]
```diff
@@ -20,5 +20,5 @@ public void deleteCampaignTest() {
 		System.out.println("execute  deleteCampaignTest");
 	}
 
-
+//THIKA AMIKAND CORRECT EDENA NODU
```
Comment (RAGHURD1) [inline src/test/java/com/crm/Campaign/CampaignTest.java:23]: OK

**Review by RAGHURD1 — COMMENTED** [review]
OK

**Comment by HARISHA05041996** [comment]
muccho bayi

**Comment by HARISHA05041996** [comment]
> 

m

**Pull request merged**